src/controllers/itemDetails/index.js

```diff
@@ -1069,6 +1072,38 @@ function enableScrollX() {
     return browser.mobile && window.screen.availWidth <= 1000;
 }
 
+function renderLyricsContainer(view, item, apiClient) {
+    const lyricContainer = view.querySelector('.lyricsContainer');
+    if (lyricContainer && item.HasLyrics) {
+        if (item.Type !== 'Audio') {
+            lyricContainer.classList.add('hide');
+            return;
+        }
+        //get lyrics
+        apiClient.ajax({
+            url: apiClient.getUrl('Audio/' + item.Id + '/Lyrics'),
+            type: 'GET',
+            dataType: 'json'
+        }).then((response) => {
+            if (!response.Lyrics) {
+                lyricContainer.classList.add('hide');
+                return;
+            }
+            lyricContainer.classList.remove('hide');
+            const itemsContainer = lyricContainer.querySelector('.itemsContainer');
+            if (itemsContainer) {
+                const html = response.Lyrics.reduce((htmlAccumulator, lyric) => {
+                    htmlAccumulator += lyric.Text + '<br/>';
```

Comment:
Escape HTML `lyric.Text`?
Unless the lyric has HTML styling, but that's dangerous.

Reply:
fixed on [a1768dd](https://github.com/jellyfin/jellyfin-web/pull/4733/commits/a1768dd4df99aaa428d8ef4ebc03c4d288e837fc)